packages/shared/src/components/sidebar/DiscoverSection.tsx

```diff
@@ -51,6 +50,40 @@ export function DiscoverSection({
     },
   ];
 
+  const completeCommentFeed = () => completeAction(ActionType.CommentFeed);
+  const discussedAction = () => {
+    completeCommentFeed();
+    onNavTabClick?.('discussed');
+  };
+
+  if (hasCommentFeed) {
+    discoverMenuItems.unshift({
+      icon: (active: boolean) => (
+        <ListIcon Icon={() => <DiscussIcon secondary={active} />} />
+      ),
+      title: 'Discussions',
+      path: '/discussed',
+      onClick: completeCommentFeed,
+      action: discussedAction,
+      ...(!hasCompletedCommentFeed && {
+        rightIcon: () => (
+          <span className="flex h-4 items-center rounded-6 bg-brand-default px-1 font-bold text-white typo-caption2">
+            NEW
+          </span>
+        ),
+      }),
+    });
+  } else {
+    discoverMenuItems.splice(2, 0, {
+      icon: (active: boolean) => (
+        <ListIcon Icon={() => <DiscussIcon secondary={active} />} />
+      ),
+      title: 'Best discussions',
+      path: '/discussed',
+      action: () => onNavTabClick?.('discussed'),
+    });
+  }
```

Comment:
lacking a bit of context here, seems we are marking the `/discussed` page as new? I don't see any modifications to the page though, or is this just to register the action?

Reply:
Correct yes, so the change is the one above.
If you're in the experiment you'll see `CommentFeed` (feed of comments) else you'll see `Feed` with most commented posts.

Reply:
nevermind, reading the PR description more carefully, I understand why this is here now 👍